<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1013</width>
    <height>505</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>460</y>
     <width>166</width>
     <height>28</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>341</width>
     <height>421</height>
    </rect>
   </property>
   <property name="title">
    <string>Shipping</string>
   </property>
   <widget class="QLineEdit" name="txtShippingCity">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>256</y>
      <width>201</width>
      <height>28</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Postcode&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>255</y>
      <width>76</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>City</string>
    </property>
    <property name="margin">
     <number>5</number>
    </property>
    <property name="indent">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Last Name</string>
    </property>
    <property name="margin">
     <number>5</number>
    </property>
    <property name="indent">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>144</y>
      <width>68</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Phone</string>
    </property>
    <property name="margin">
     <number>5</number>
    </property>
    <property name="indent">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>218</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Address 2</string>
    </property>
    <property name="margin">
     <number>5</number>
    </property>
    <property name="indent">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>33</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>First Name</string>
    </property>
    <property name="margin">
     <number>5</number>
    </property>
    <property name="indent">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>322</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>ZipCode</string>
    </property>
    <property name="margin">
     <number>5</number>
    </property>
    <property name="indent">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>359</y>
      <width>68</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Country</string>
    </property>
    <property name="margin">
     <number>5</number>
    </property>
    <property name="indent">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtShippingLastName">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>71</y>
      <width>201</width>
      <height>28</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Email address&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>181</y>
      <width>81</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Address 1</string>
    </property>
    <property name="margin">
     <number>5</number>
    </property>
    <property name="indent">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtShippingAddress2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>219</y>
      <width>201</width>
      <height>28</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Your city&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtShippingAddress1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>182</y>
      <width>201</width>
      <height>28</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Address 2&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtShippingFirstName">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>34</y>
      <width>201</width>
      <height>28</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Profile name&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtsShippingPhone">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>145</y>
      <width>201</width>
      <height>28</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Main address&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>107</y>
      <width>56</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
    <property name="margin">
     <number>5</number>
    </property>
    <property name="indent">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtShippingEmail">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>108</y>
      <width>201</width>
      <height>28</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Telephone number&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtShippingZipcode">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>323</y>
      <width>201</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cbShippingCountry">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>360</y>
      <width>201</width>
      <height>28</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>United States</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Canada</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_26">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>State</string>
    </property>
    <property name="margin">
     <number>5</number>
    </property>
    <property name="indent">
     <number>5</number>
    </property>
   </widget>
   <widget class="QComboBox" name="cbShippingState">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>290</y>
      <width>201</width>
      <height>28</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>GA</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AL</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AK</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>50</y>
     <width>291</width>
     <height>371</height>
    </rect>
   </property>
   <property name="title">
    <string>Payment</string>
   </property>
   <widget class="QLineEdit" name="txtCardNumber">
    <property name="geometry">
     <rect>
      <x>117</x>
      <y>136</y>
      <width>161</width>
      <height>28</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Card number&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>61</y>
      <width>79</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Card type</string>
    </property>
    <property name="margin">
     <number>5</number>
    </property>
    <property name="indent">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtCvv">
    <property name="geometry">
     <rect>
      <x>117</x>
      <y>173</y>
      <width>161</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cbExpiryMonth">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>210</y>
      <width>61</width>
      <height>28</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>01</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>02</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>03</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>04</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>05</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>06</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>07</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>07</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>08</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>09</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>10</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>11</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>12</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>209</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Expiry Date</string>
    </property>
    <property name="margin">
     <number>5</number>
    </property>
    <property name="indent">
     <number>5</number>
    </property>
   </widget>
   <widget class="QComboBox" name="cbCardType">
    <property name="geometry">
     <rect>
      <x>117</x>
      <y>62</y>
      <width>161</width>
      <height>28</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Card type&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <item>
     <property name="text">
      <string>Mastercard</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Visa</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>American Express</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Discover</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="txtCardName">
    <property name="geometry">
     <rect>
      <x>117</x>
      <y>99</y>
      <width>161</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>98</y>
      <width>88</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Card Name</string>
    </property>
    <property name="margin">
     <number>5</number>
    </property>
    <property name="indent">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>135</y>
      <width>104</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Card Number</string>
    </property>
    <property name="margin">
     <number>5</number>
    </property>
    <property name="indent">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>172</y>
      <width>44</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>CVV</string>
    </property>
    <property name="margin">
     <number>5</number>
    </property>
    <property name="indent">
     <number>5</number>
    </property>
   </widget>
   <widget class="QComboBox" name="cbExpiryYear">
    <property name="geometry">
     <rect>
      <x>207</x>
      <y>210</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <item>
     <property name="text">
      <string>2021</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2022</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2023</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2024</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2025</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2026</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2027</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2028</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2029</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2030</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>209</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>/</string>
    </property>
    <property name="margin">
     <number>5</number>
    </property>
    <property name="indent">
     <number>5</number>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>50</y>
     <width>341</width>
     <height>421</height>
    </rect>
   </property>
   <property name="title">
    <string>Billing</string>
   </property>
   <widget class="QLineEdit" name="txtBillingCity">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>256</y>
      <width>201</width>
      <height>28</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Postcode&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>255</y>
      <width>76</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>City</string>
    </property>
    <property name="margin">
     <number>5</number>
    </property>
    <property name="indent">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>101</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Last Name</string>
    </property>
    <property name="margin">
     <number>5</number>
    </property>
    <property name="indent">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_17">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>144</y>
      <width>68</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Phone</string>
    </property>
    <property name="margin">
     <number>5</number>
    </property>
    <property name="indent">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_18">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>218</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Address 2</string>
    </property>
    <property name="margin">
     <number>5</number>
    </property>
    <property name="indent">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_19">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>33</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="text">
     <string>First Name</string>
    </property>
    <property name="margin">
     <number>5</number>
    </property>
    <property name="indent">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_20">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>319</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>ZipCode</string>
    </property>
    <property name="margin">
     <number>5</number>
    </property>
    <property name="indent">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>356</y>
      <width>68</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Country</string>
    </property>
    <property name="margin">
     <number>5</number>
    </property>
    <property name="indent">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtBillingLastName">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>71</y>
      <width>201</width>
      <height>28</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Email address&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_22">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>181</y>
      <width>81</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Address 1</string>
    </property>
    <property name="margin">
     <number>5</number>
    </property>
    <property name="indent">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtBillingAddress2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>219</y>
      <width>201</width>
      <height>28</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Your city&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtBillingAddress1">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>182</y>
      <width>201</width>
      <height>28</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Address 2&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtBillingFirstName">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>34</y>
      <width>201</width>
      <height>28</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Profile name&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtsBillingPhone">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>145</y>
      <width>201</width>
      <height>28</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Main address&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_23">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>107</y>
      <width>56</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Email</string>
    </property>
    <property name="margin">
     <number>5</number>
    </property>
    <property name="indent">
     <number>5</number>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtBillingEmail">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>108</y>
      <width>201</width>
      <height>28</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Telephone number&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtBillingZipcode">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>320</y>
      <width>201</width>
      <height>28</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="cbBillingCountry">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>357</y>
      <width>201</width>
      <height>28</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>United States</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Canada</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="cbBillingState">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>290</y>
      <width>201</width>
      <height>28</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>GA</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AL</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>AK</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_27">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>91</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>State</string>
    </property>
    <property name="margin">
     <number>5</number>
    </property>
    <property name="indent">
     <number>5</number>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Profile name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtProfileName">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>251</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbSameAsShipping">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>10</y>
     <width>331</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Same as shipping</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>txtShippingFirstName</tabstop>
  <tabstop>txtShippingLastName</tabstop>
  <tabstop>txtShippingEmail</tabstop>
  <tabstop>txtsShippingPhone</tabstop>
  <tabstop>txtShippingAddress1</tabstop>
  <tabstop>txtShippingAddress2</tabstop>
  <tabstop>txtShippingCity</tabstop>
  <tabstop>txtShippingZipcode</tabstop>
  <tabstop>cbCardType</tabstop>
  <tabstop>txtCardName</tabstop>
  <tabstop>txtCardNumber</tabstop>
  <tabstop>cbExpiryMonth</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
